x/upgrade/keeper/keeper.go

```diff
@@ -243,24 +243,18 @@ func (k Keeper) GetLastCompletedUpgrade(ctx sdk.Context) (string, int64) {
 	iter := sdk.KVStoreReversePrefixIterator(ctx.KVStore(k.storeKey), []byte{types.DoneByte})
 	defer iter.Close()
 
-	var upgrades []upgrade
+	var latest upgrade
+	var found bool
```

Comment:
```suggestion
	var (
	latest upgrade
	 found bool
	 )
```

Reply:
honestly I prefer simple var rather than var log

Reply:
Consistency above all else, please group these.

Reply:
That is the standard -- hence why I added the comment. Anytime I see these multi-line `var` statements, I always comment on the PR to group them. Please group them :)

Reply:
I already did that 4h ago ;) [fde59da](https://github.com/cosmos/cosmos-sdk/pull/12268/commits/fde59dae3d03b7a87cf7768ae0828c309ef6f5de)